cellx/networks/classifier.py

```diff
@@ -1 +1,13 @@
-# this will be the classifier
+# Building the CNN classifier:
+
+import tensorflow as tf
+from tensorflow import keras as K
+
+from cellx import layers
+
+input = K.Input(shape=(2,))
+encoder = layers.Encoder2D()(input)
+dense = K.layers.Dense(512, activation="relu")(encoder)
+output = K.layers.Dense(4, activation=tf.nn.softmax)(dense)
+
+classifier_model = K.Model(inputs=input, outputs=output)
```

Comment:
I think we should make this a function for now (and transition to a full model later), e.g.:
```python
def build_classifier(shape: tuple, outputs: int, dense_filters: int = 512):
    """add docstrings :)"""
    input = K.Input(shape=shape)
    ...
    return classifier_model
```